cmd/cmd.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/spf13/viper"
 	"github.com/wagoodman/go-partybus"
 
+	iface "github.com/anchore/go-logger"
```

Comment:
nit: maybe something with `log` or `logger` in the name? like `loggerAPI`? to make it a tiny bit more clear below

Reply:
Updated to `anchoreLogger` since `log`, `logger`, and other names were taken

Reply:
Updated to anchoreLogger since log, logger, and other names were taken